diesel/src/pg/connection/raw.rs

```diff
@@ -132,6 +132,12 @@ impl RawResult {
     pub fn as_ptr(&self) -> *mut PGresult {
         self.0
     }
+
+    pub fn error_message(&self) -> &str {
+        let ptr = unsafe { PQresultErrorMessage(self.0) };
```

Comment:
This pointer is never null in any case? Otherwise it may be better to add a additional check for this. I think it may be better to add the additional check for null here because in theory this could be null…

Reply:
The pointer is never null in any case here